app/components/CalendarRefresh.js

```diff
@@ -0,0 +1,18 @@
+const React = require('react');
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).